lib/srv/db/common/errors.go

```diff
@@ -111,7 +111,8 @@ func ConvertConnectError(err error, sessionCtx *Session) error {
 	switch {
 	case strings.Contains(errString, "x509: certificate signed by unknown authority"):
 		return trace.AccessDenied("Database service cannot validate database's certificate: %v. Please verify if the correct CA bundle is used in the database config.", err)
-
+	case strings.Contains(errString, "x509: certificate has expired or is not yet valid"):
+		return trace.AccessDenied("Database service cannot validate the database's certificate: %s. Please verify if the database certificate is outdated and the database service time and date are set correctly.", err)
```

Comment:
I wonder if we can put a new line char in error message.  Right now we will print long string where the most important part of information is put at the end of the string. 

Also the following text: 
> Please verify if the database certificate is outdated and the database service time and date are set correctly

In context of ordinary user can be misleading by saying that database certificate is outdated a user probably will check their certs it `./tsh/.../db/`  directory. 
Could we give a user more context and redirect to our doc with following message: 
"..., please contact your administrator and ask them to follow docs ref"

Reply:
> I wonder if we can put a new line char in error message. Right now we will print long string where the most important part of information is put at the end of the string.

I'm not sure if we do this in other cases. Letting the user's terminal do this seems better since it can fit long texts better.

> Could we give a user more context and redirect to our doc with following message

In that case, we would direct them to the database guides, right?

Reply:
Yea, I was thinking about creating something like troubleshooting -  https://goteleport.com/docs/desktop-access/troubleshooting/ for database access  where we can point a user in case of know error.

Reply:
The troubleshooting page idea seems a better approach here. The error message will be more generic, but the page can have a more accurate and complete solution.

I've updated the PR to have the link and added the page.